NuKeeper/Configuration/CommandLineArguments.cs

```diff
@@ -8,7 +9,8 @@ public class CommandLineArguments
         [CommandLine("mode", "m"), Required]
         public string Mode;
 
-        [CommandLine("github_token", "t"), Required, SensitiveInformation]
+        [Environment("NuKeeper_github_token"), Required, SensitiveInformation]
+        [OverriddenBy(ConfigurationSources.CommandLine, "t")]
```

Comment:
I thought it would be very verbose to include a full name AND an alias for the override (not to mention, certain sources are better without an alias e.g. environment or config file)